partiql-parser/src/test/kotlin/org/partiql/parser/internal/PartiQLParserDDLTests.kt

```diff
@@ -312,6 +313,220 @@ class PartiQLParserDDLTests {
+                    tableDefinition(
+                        listOf(
+                            tableDefinitionAttribute(
+                                identifierSymbol("a", Identifier.CaseSensitivity.INSENSITIVE),
+                                Type.Struct(
+                                    emptyList()
+                                ),
+                                emptyList(),
+                            )
+                        ),
+                        emptyList()
+                    ),
+                )
+            ),
+
+            SuccessTestCase(
+                "CREATE TABLE with ARRAY",
+                """
+                    CREATE TABLE tbl (
+                        a ARRAY<INT2>
+                    )
+                """.trimIndent(),
+                ddlOpCreateTable(
+                    identifierSymbol("tbl", Identifier.CaseSensitivity.INSENSITIVE),
+                    tableDefinition(
+                        listOf(
+                            tableDefinitionAttribute(
+                                identifierSymbol("a", Identifier.CaseSensitivity.INSENSITIVE),
+                                Type.Array(Type.Int2()),
+                                emptyList(),
+                            )
+                        ),
+                        emptyList()
+                    ),
+                )
+            ),
+            SuccessTestCase(
+                "CREATE TABLE with ARRAY of Struct",
+                """
+                    CREATE TABLE tbl (
+                        a ARRAY< STRUCT< b:INT2 > >
+                    )
+                """.trimIndent(),
+                ddlOpCreateTable(
+                    identifierSymbol("tbl", Identifier.CaseSensitivity.INSENSITIVE),
+                    tableDefinition(
+                        listOf(
+                            tableDefinitionAttribute(
+                                identifierSymbol("a", Identifier.CaseSensitivity.INSENSITIVE),
+                                Type.Array(
+                                    Type.Struct(
+                                        listOf(
+                                            Type.Struct.Field(
+                                                identifierSymbol("b", Identifier.CaseSensitivity.INSENSITIVE),
+                                                Type.Int2(),
+                                                emptyList()
+                                            ),
+                                        )
+                                    ),
+                                ),
+                                emptyList(),
+                            )
+                        ),
+                        emptyList()
+                    ),
+                )
+            ),
+            SuccessTestCase(
+                "CREATE TABLE no space between angle right",
+                """
+                    CREATE TABLE tbl(
+                        a LIST<STRUCT<b:INT2>>
```

Comment:
nice that the double right angle parses now. should this use `ARRAY` since `LIST` will be deprecated/removed?

Reply:
I think we still permit parsing of LIST type, it is just that the AST node we use for LIST/ARRAY should be the same.